std/datetime/date.d

```diff
@@ -3055,20 +3055,19 @@ public:
         if (isSomeString!S)
     {
         import std.algorithm.searching : countUntil;
-        import std.conv : to;
         import std.exception : enforce;
         import std.format : format;
         import std.string : strip;
 
-        immutable dstr = to!dstring(strip(isoString));
+        immutable str = strip(isoString);
```

Comment:
@jmdavis None of the functions here use auto decoding. I've already remove it from `Date.fromISOString` and `TimeOfDay.fromISOString`, and `countUntil` with a single needle forwards to `find`, which already deals with auto-decoding.